Scripts/Python/ki/xKIChat.py

```diff
@@ -1217,6 +1217,23 @@ def ChangePassword(self, newPassword):
             return
         PtChangePassword(newPassword)
 
+    ## Update the Chronicle's KI Text Color values.
+    # This uses an RGB triplet with parts separated by "," or an empty string for the default chat color.
+    def UpdateTextColorChronicle(self):
+
+        vault = ptVault()
+        entry = vault.findChronicleEntry(kChronicleKITextColor)
+        newVal = ""
+        if isinstance(self.chatMgr.chatTextColor, ptColor):
+            newVal = f"{self.chatMgr.chatTextColor.getRed()},{self.chatMgr.chatTextColor.getGreen()},{self.chatMgr.chatTextColor.getBlue()}"
+
+        PtDebugPrint(f"xKIChat.UpdateTextColorChronicle(): Setting KI Text Color chronicle to: \"{newVal}\".", level=kWarningLevel)
+        if entry is not None:
+            entry.chronicleSetValue(newVal)
+            entry.save()
+        else:
+            vault.addChronicleEntry(kChronicleKITextColor, kChronicleKITextColorType, newVal)
```

Comment:
```suggestion
        newVal = ""
        if isinstance(self.chatMgr.chatTextColor, ptColor):
            newVal = f"{self.chatMgr.chatTextColor.getRed()},{self.chatMgr.chatTextColor.getGreen()},{self.chatMgr.chatTextColor.getBlue()}"

        PtDebugPrint(f"xKIChat.UpdateTextColorChronicle(): Setting KI Text Color chronicle to: \"{newVal}\".", level=kWarningLevel)
        vault.addChronicleEntry(kChronicleKITextColor, kChronicleKITextColorType, newVal)
```

`addChronicleEntry()` updates any already existing entry.

Reply:
Ooooooh... I, uh, kind of copied that if/else pattern from somewhere else so I didn't know that. Apparently that usage is poor in a lot of places. 😬

Reply:
It's possible that some older variant of Uru did *not* do that check, but MOUL does.